dashboards/grafana-dashboard-insights-rosocp-general.configmap.yaml

```diff
@@ -86,7 +86,7 @@ data:
                 "uid": "${datasource}"
               },
               "editorMode": "code",
-              "expr": "sum(kafka_consumergroup_group_lag{topic=~\".*hccm.ros.events\", group=\"ros-ocp\"})",
+              "expr": "sum(aws_kafka_sum_offset_lag_sum{topic=~\".*hccm.ros.events\", consumer_group=\"ros-ocp\"})",
```

Comment:
```
"datasource": {
                "uid": "${datasource}"
              },
```

Lately noticed that, this is currently using `datasource` variable  defined here - https://github.com/RedHatInsights/ros-ocp-backend/blob/main/dashboards/grafana-dashboard-insights-rosocp-general.configmap.yaml#L1272

With an email reference, this new metric is provided by the Cloudwatch.  In my opinion, you might need to define additional template variable. Could you please also check on this and get it confirm.
Thanks!

Reply:
The data source seems to be actually different for cloudwatch metrics.
`"regex": "/aws-resources-exporter-(production|stage)/",`
Changes tested on a new dashboard in grafana, and pushed. Link available in the JIRA card.